docs/user-manuals/workloadspread.md

```diff
@@ -204,7 +223,95 @@ The workload managed by WorkloadSpread will scale according to the defined order
 # deletion order: b -> a -> c
 ```
 
+## Use WorkloadSpread with customized workload
+
+If you want to use WorkloadSpread with custom workloads, which is disabled by default, some
```

Comment:
Please explain the customized workload must support scale sub-resources

Reply:
scale sub-resources seems not to be used